test/models/eis/QuoteFormatterSpec.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.eis
+
+import base.SpecBase
+import org.scalatest.MustMatchers
+
+class QuoteFormatterSpec extends SpecBase with MustMatchers {
```

Comment:
I love this test.

Please add a couple of tests with leading and trailing white space, as that is part of the regex.

I know they will pass, but may help future dev

Reply:
Good shout.   I hadn't spotted that in the regex - it's not explicitly called out in the ticket.    So the "formatting" should kick in for space followed by illegal character at the beginning or illegal character followed by space at the end.   Agreed?

Reply:
Agree exactly.